stdlib/experimental/table/fill_test.flux

```diff
@@ -0,0 +1,127 @@
+package table_test
+
+
+import "planner"
+import "testing"
+import "testing/expect"
+import "experimental/table"
+
+inData = "
+#datatype,string,long,dateTime:RFC3339,string,string,double,string
+#group,false,false,false,true,true,false,true
+#default,_result,,,,,,
+,result,table,_time,_measurement,_field,_value,t0
+,,0,2021-04-13T09:00:00Z,m0,f0,2.0,a
+,,0,2021-04-13T09:05:00Z,m0,f0,9.0,a
+,,0,2021-04-13T09:15:00Z,m0,f0,7.0,a
+,,0,2021-04-13T09:25:00Z,m0,f0,3.0,a
+,,0,2021-04-13T09:45:00Z,m0,f0,5.0,a
+,,0,2021-04-13T09:55:00Z,m0,f0,1.0,a
+,,1,2021-04-13T09:05:00Z,m0,f0,4.0,b
+,,1,2021-04-13T09:10:00Z,m0,f0,1.0,b
+,,1,2021-04-13T09:15:00Z,m0,f0,2.0,b
+,,1,2021-04-13T09:30:00Z,m0,f0,5.0,b
+,,1,2021-04-13T09:35:00Z,m0,f0,6.0,b
+,,1,2021-04-13T09:40:00Z,m0,f0,8.0,b
+"
+loadData = () => testing.loadStorage(csv: inData)
+    |> range(start: 2021-04-13T09:00:00Z, stop: 2021-04-13T10:00:00Z)
+
+testcase window {
+    want = testing.loadMem(
```

Comment:
I hate to ask this, because it's so new, but it'd be nice if we could just use `array.from` and `testing.load` here. What do you think about making that change?

Reply:
Let me give it a try.